temporal-normalizer/src/main/java/edu/illinois/cs/cogcomp/temporal/normalizer/main/TemporalChunkerAnnotator.java

```diff
@@ -0,0 +1,845 @@
+/**
```

Comment:
One more suggested change: 
Go to the [`PipelineFactory`](https://github.com/CogComp/cogcomp-nlp/blob/master/pipeline/src/main/java/edu/illinois/cs/cogcomp/pipeline/main/PipelineFactory.java) and add this annotator there. This way we will have it easily accessible from the pipeline.